lib/skate_web/controllers/detours_controller.ex

```diff
@@ -11,15 +11,11 @@ defmodule SkateWeb.DetoursController do
   alias Util.Location
 
   @spec update_snapshot(Plug.Conn.t(), map()) :: Plug.Conn.t()
-  def update_snapshot(conn, %{"snapshot" => %{"context" => context} = snapshot}) do
-    uuid = Map.get(context, "uuid")
-
+  def update_snapshot(conn, %{"snapshot" => snapshot}) do
     %{id: user_id} = AuthManager.Plug.current_resource(conn)
 
     {:ok, %Skate.Detours.Db.Detour{id: returned_uuid}} =
-      Detours.update_or_create_detour_for_user(user_id, uuid, %{
-        state: snapshot
-      })
```

Comment:
I wonder why I previously wrapped the snapshot like this? I see that your changes assign state properly in the db, so... great! Much simpler 😊